recipes/newrelic/elasticsearch/suse.yml

```diff
@@ -0,0 +1,124 @@
+    default: 9200
+  - name: "NR_CLI_CONFIG_PATH"
+    prompt: "Please provide the path to ElasticSearch config file, if none is provided then the default: /etc/elasticsearch/elasticsearch.yml will be used."
+    default: "/etc/elasticsearch/elasticsearch.yml"
+  - name: "NR_CLI_API_USE_SSL"
+    prompt: "Do you want to use SSL to connect?"
+    default: false
+  - name: "NR_CLI_API_CA_BUNDLE_DIR"
+    prompt: "Please enter the directory in which the SSL certificate is located on the host. If using SSL please leave this field empty."
+  - name: "NR_CLI_API_CA_BUNDLE_FILE"
+    prompt: "Please enter the path to the SSL certificate file on the host. If using SSL please leave this field empty."
+
+install:
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: setup
+
+    setup:
+      label: "Installing elasticsearch integration..."
+      cmds:
+        - |
+          sudo mkdir -p "/etc/newrelic-infra/integrations.d"
+        - |
+          sudo zypper -n update 
+        - |
+          sudo zypper -n install nri-elasticsearch
+        - |
+          if [ -f /etc/newrelic-infra/integrations.d/elasticsearch-config.yml ]; then
+            sudo rm /etc/newrelic-infra/integrations.d/elasticsearch-config.yml;
+          fi
+
+          sudo touch /etc/newrelic-infra/integrations.d/elasticsearch-config.yml;
+        - |
+          if [ "$NR_CLI_API_USE_SSL" == false ]; then
+            sudo tee -a /etc/newrelic-infra/integrations.d/elasticsearch-config.yml > /dev/null <<"EOT"
+          integration_name: com.newrelic.elasticsearch
+
+          instances:
+            - name: elasticsearch
+              command: all
+              arguments:
+                cluster_environment: staging
+                config_path: {{.NR_CLI_API_CONFIG_PATH}}
+                hostname: {{.NR_CLI_API_HOSTNAME}}
+                local_hostname: localhost
+                username: {{.NR_CLI_API_USERNAME}}
+                password: {{.NR_CLI_API_PASSWORD}}
+                port: {{.NR_CLI_API_PORT}}
+                timeout: 30
+                use_ssl: false
+                ca_bundle_dir: 
+                ca_bundle_file: 
+                indices_regex:
+                collect_indices: true
+                collect_primaries: true
+          EOT
+          else
+            sudo tee -a /etc/newrelic-infra/integrations.d/elasticsearch-config.yml > /dev/null <<"EOT"
+          integration_name: com.newrelic.elasticsearch
+
+          instances:
+            - name: elasticsearch
+              command: all
+              arguments:
+                cluster_environment: staging
+                config_path: {{.NR_CLI_API_CONFIG_PATH}}
+                hostname: {{.NR_CLI_API_HOSTNAME}}
+                local_hostname: localhost
+                username: {{.NR_CLI_API_USERNAME}}
+                password: {{.NR_CLI_API_PASSWORD}}
+                port: {{.NR_CLI_API_PORT}}
+                timeout: 30
+                use_ssl: true
```

Comment:
@JakubKotkowiak I believe we should be using the input param NR_CLI_API_USE_SSL here.